appium/tests/helpers/ElementHelper.ts

```diff
@@ -49,10 +49,12 @@ class ElementHelper {
     await element.doubleClick();
   }
 
-  static waitAndClick = async (element: WebdriverIO.Element, delayMs = 50) => {
-    await element.waitForDisplayed();
+  static waitAndClick = async (element: WebdriverIO.Element, delayMs?: number) => {
+    await this.waitElementVisible(element);
     // stability fix to make sure element is ready for interaction
-    await browser.pause(delayMs);
+    if (delayMs !== undefined) {
+      await browser.pause(delayMs);
+    }
     await element.click();
   }
```

Comment:
This will likely make tests more brittle, I believe waiting 50ms by default was helping them run more smoothly.

If you want to have an option to skip this, I recommend to use `if (delayMs) {`. Then the argument doesn't have to be optional, and you can set `waitAndClick(xxx, 0)` for the one case you explicitly don't want to wait.

Reply:
During my testing it worked well - probably `waitElementVisible(element)` is enough for element waiting.
I'll add `50ms` delay back to be sure it won't fail in the future.

Reply:
`50ms` should be ok - it adds just 2 seconds for total tests duration